curvefs/src/metaserver/storage/rocksdb_storage.cpp

```diff
@@ -451,8 +391,6 @@ Status RocksDBStorage::Commit() {
     if (!s.ok()) {
         LOG(ERROR) << "RocksDBStorage commit transaction failed"
                    << ", status=" << s.ToString();
-    } else {
-        CommitKeys();
```

Comment:
no need commit or rollback ？

Reply:
> no need commit or rollback ？

yep~~ it only commit or rollback keys to container, now we remove these all.